AsyncDisplayKit/Details/_ASDisplayView.mm

```diff
@@ -202,6 +202,15 @@ - (void)willRemoveSubview:(UIView *)subview
 #endif
 }
 
+- (CGSize)sizeThatFits:(CGSize)size
```

Comment:
Correct me if I'm wrong, but it seems this method doesn't immediately change the size of this node, but eventually it will (during the next layout pass). If it's the case, this is a bit troublesome. Maybe we can add a flag to `-layoutThatFits:` to indicate "don't store the new layout as pending, I'm just testing the water". 

Btw, do we need to support `sizeToFits` as well?